examples/annotation/revise_news_classification_prompt.yaml

```
@@ -0,0 +1,20 @@
+    Guidelines:
+    - The verifier assessment provides expert reasoning about potential classification issues.
+    - If the verifier identified problems with the initial prediction, correct them.
+    - If the verifier confirmed the prediction is reasonable, maintain it.
+    - Only use the categories provided in the guidelines, the exact label that you think is most appropriate should be used.
+
+    Here is the query for annotation:
+
+    Text: {{text}}
+    Initial Prediction: {{output}}
+    Verifier Assessment: {{assessment_output}}
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)